pkg/networkmanager/firewall.jsx

```diff
@@ -204,9 +206,9 @@ class AddServicesModal extends React.Component {
     save() {
         if (this.state.custom) {
             firewall.createService(this.state.custom_id, this.state.custom_name, this.createPorts())
-                    .then(firewall.enableService(this.state.custom_id));
+                    .then(firewall.enableService(this.state.zones.concat(), this.state.custom_id));
```

Comment:
These functions shouldn't modify their arguments, so copying the list should not be necessary